plugins/GSdx/GSRendererDX.cpp

```diff
@@ -48,15 +48,139 @@ GSRendererDX::~GSRendererDX()
 {
 }
 
+void GSRendererDX::EmulateAtst(const int pass, const GSTextureCache::Source* tex)
+{
+	GSDeviceDX* dev = (GSDeviceDX*)m_dev;
```

Comment:
For the record I do that in openGL because it allow the compiler to optimize virtual function call. OpenGL device is the lastest inheritance, so you know the function at compile time. On dx it is useless you still need to dispatch it at runtime to select either dx9 or dx11.